src/Compilers/Core/Portable/PortableShim.cs

```diff
@@ -413,6 +413,11 @@ internal static class Encoding
                 .GetTypeInfo()
                 .GetDeclaredMethod(nameof(GetEncoding), paramTypes: new[] { typeof(int) })
                 .CreateDelegate<Func<int, System.Text.Encoding>>();
+
+            internal static readonly Func<System.Text.Encoding> Default = Type
```

Comment:
GetEncoding exists on both Desktop and CoreCLR, but Encoding.Default does not exist on CoreCLR.  We have to change this code to use a reflection-gate on whether or not we're running on CoreCLR, then either reflection load the encoding from the desktop, or register a CodePagesEncodingProvider from System.Text.Encoding.CodePages using Encoding.RegisterProvider.